src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowsOutWalker.cs

```diff
@@ -106,7 +106,7 @@ private Symbol GetNodeSymbol(BoundNode node)
                 {
                     case BoundKind.DeclarationPattern:
                         {
-                            return ((BoundDeclarationPattern)node).LocalSymbol;
+                            return ((BoundDeclarationPattern)node).Variable as LocalSymbol;
```

Comment:
VariableAccess isn't a symbol, and doesn't represent the logical symbol represented by this node.

---
In reply to: [80798952](https://github.com/dotnet/roslyn/pull/13999#discussion_r80798952) [](ancestors = 80798952)